### PR TITLE
[ID] Add `Ranking_criteria/osu!catch`

### PR DESCRIPTION
**Pull request by mousesalt**
new translation Indonesia for ranking criteria osu!catch here.
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!catch/id.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Ranking criteria osu!catch
+
+***Perhatian: Artikel ini merupakan perpanjangan dari [ranking criteria umum](/wiki/Ranking_Criteria).***
+
+**Ranking criteria osu!catch** menjabarkan [aturan dan pedoman](/wiki/Ranking_Criteria#general-terms) spesifik untuk [beatmaps](/wiki/Beatmap) [osu!catch](/wiki/Game_mode/osu!catch)  yang wajib diikuti untuk tahapan dari [Prosedur ranking beatmap](/wiki/Beatmap_ranking_procedure).
```
Comment (hikikuman) [inline wiki/Ranking_Criteria/osu!catch/id.md:5]:
```suggestion
**Ranking criteria osu!catch** menjabarkan [aturan dan pedoman](/wiki/Ranking_Criteria#general-terms) spesifik untuk [beatmap](/wiki/Beatmap) [osu!catch](/wiki/Game_mode/osu!catch) yang wajib diikuti untuk tahapan dari [Prosedur ranking beatmap](/wiki/Beatmap_ranking_procedure).
```

File: wiki/Ranking_Criteria/osu!catch/id.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+## Keseluruhan
+
+Keseluruhan aturan dan pedoman yang berlaku untuk setiap jenis kesulitan osu!catch. Aturan dan pedoman ritme berlaku untuk beatmap sekitar 180 BPM dengan 4/4 time signature. Jika lagu secara drastis lebih cepat atau lebih lambat, beberapa variabel mungkin berbeda, seperti yang dijelaskan dalam [Penskalaan BPM di kriteria ranking](/wiki/Ranking_Criteria/Scaling_BPM).
```
Comment (hikikuman) [inline wiki/Ranking_Criteria/osu!catch/id.md:9]:
```suggestion
Keseluruhan aturan dan pedoman berlaku untuk setiap jenis tingkat kesulitan osu!catch. Aturan dan pedoman ritme berlaku untuk beatmap sekitar 180 BPM dengan 4/4 time signature. Jika lagu secara drastis lebih cepat atau lebih lambat, beberapa variabel mungkin berbeda, seperti yang dijelaskan dalam [Penskalaan BPM di ranking criteria](/wiki/Ranking_Criteria/Scaling_BPM).
```

File: wiki/Ranking_Criteria/osu!catch/id.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+- **Slider tail dari slider yang diperpanjang harus dipotong sesuai dengan struktur ketukan lagu.** Jika lagunya menggunakan beat lurus, 1/4, 1/8, dan 1/16 harus digunakan. Jika lagunya menggunakan ketukan ayunan, 1/6 or 1/12 harus digunakan. Jika lagu memiliki suara snap yang berbeda dari yang direkomendasikan, snap ke beat yang sebenarnya selalu diprioritaskan.
+- **[Dash](/wiki/Gameplay/Dash) dan [hyperdash](/wiki/Gameplay/Hyperdash) tidak boleh digunakan ketika tujuan dari dash atau hyperdash terletak di dekat batas kiri atau kanan area permainan.** Ini menciptakan gerakan yang tidak nyaman karena penangkap dihentikan secara paksa saat mencapai batas area bermain (x:16 di kiri dan x:496 di kanan).
+- **Pastikan [kombo](/wiki/Beatmapping/Combo) tidak mencapai panjang yang tidak masuk akal.** Menangkap [fruit](/wiki/Hit_object/Fruit) akan menumpuk di piring dan berpotensi menghalangi pandangan pemain. Ingatlah bahwa slider tail, repeat dan [spinner](/wiki/Hit_object/Spinner) [pisang](/wiki/Hit_object/Banana) juga dihitung sebagai "fruit". Kombo baru harus ditempatkan secara teratur untuk membersihkan piring dan menghindari hal ini terjadi.
```
Comment (hikikuman) [inline wiki/Ranking_Criteria/osu!catch/id.md:24]:
```suggestion
- **Pastikan [kombo](/wiki/Beatmapping/Combo) tidak mencapai panjang yang tidak masuk akal.** Menangkap [fruit](/wiki/Hit_object/Fruit) akan menumpuk di piring dan berpotensi menghalangi pandangan pemain. Ingatlah bahwa slider tail, repeat dan [spinner](/wiki/Hit_object/Spinner) [banana](/wiki/Hit_object/Banana) juga dihitung sebagai "fruit". Kombo baru harus ditempatkan secara teratur untuk membersihkan piring dan menghindari hal ini terjadi.
```
kayanya ini harusnya ditulis banana juga sih. kalo mau keep, artinya "fruit" juga harus jadi "buah". gimana?

File: wiki/Ranking_Criteria/osu!catch/id.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+- [Circle Size](/wiki/Beatmapping/Circle_size) harus di antara 2.5 d 3.5.
+
+### ![](/wiki/shared/diff/hard-c.png?20211215) Platter
+
+#### Rules
+
+- **[Hyperdash](/wiki/Gameplay/Hyperdash) gap wajib setidaknya 125 ms di antara dua objek.**
+- **[Hyperdash](/wiki/Gameplay/Hyperdash) tidak dapat digunakan pada [drop](/wiki/Hit_object/Juice_stream#drop) individu dan/atau perulangan slider.** Akurasi dan kontrol yang diperlukan akan tidak masuk akal pada level ini dan dapat menciptakan situasi di mana pemain berpotensi gagal membaca jalur slider.
+- **[Hyperdash](/wiki/Gameplay/Hyperdash) dari snap beat yang berbeda tidak boleh digunakan di antara [fruit](/wiki/Hit_object/Fruit) secara berurutan.** Contohnya, hyperdash 1/2 diikuti hyperdash 1/4.
+- **[Hyperdash](/wiki/Gameplay/Hyperdash) yang [basic-snapped](/wiki/Gameplay/Dash_snapping#basic-snapped) tidak boleh digunakan lebih dari dua kali antara [fruits](/wiki/Hit_object/Fruit) secara berturut-turut.**
```
Comment (hikikuman) [inline wiki/Ranking_Criteria/osu!catch/id.md:119]:
```suggestion
- **[Hyperdash](/wiki/Gameplay/Hyperdash) yang [basic-snapped](/wiki/Gameplay/Dash_snapping#basic-snapped) tidak boleh digunakan lebih dari dua kali antara [fruit](/wiki/Hit_object/Fruit) secara berturut-turut.**
```

File: wiki/Ranking_Criteria/osu!catch/id.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+- **[Edge dash](/wiki/Gameplay/Edge_dash) tidak boleh digunakan.** Mereka membutuhkan waktu yang sangat tepat, yang tidak dapat diharapkan dari pemain yang kurang berpengalaman.
+- **Setidaknya 250 ms wajib dibiarkan di antara circle/slider dan awal dan akhir dari [spinner](/wiki/Hit_object/Spinner).** Ini untuk memastikan keterbacaan.
+
+#### Pedoman
+
+- **Semua jarak harus jelas apakah mereka mengharuskan pemain untuk [walk](/wiki/Gameplay/Walk) atau [dash](/wiki/Gameplay/Dash).** Hal ini untuk memastikan bahwa pemain dapat dengan mudah mengenali pola yang membutuhkan dash.
+- **[Dash](/wiki/Gameplay/Dash) yang [basic-snapped](/wiki/Gameplay/Dash_snapping#basic-snapped) tidak boleh digunakan secara berurutan saat menggunakan ketukan snap yang berbeda.** Contohnya, dash 1/1 diikuti oleh dash 1/2.
+- **[Dash](/wiki/Gameplay/Dash) yang [higher-snapped](/wiki/Gameplay/Dash_snapping#higher-snapped) tidak harus diikuti oleh pattern [antiflow](/wiki/Beatmapping/Antiflow).**
+- **[Kombo](/wiki/Beatmapping/Combo) tidak boleh melebihi 10 objek termasuk slider tail dan pengulangan.** [Spinners](/wiki/Hit_object/Spinner) adalah pengecualian.
```
Comment (hikikuman) [inline wiki/Ranking_Criteria/osu!catch/id.md:103]:
```suggestion
- **[Kombo](/wiki/Beatmapping/Combo) tidak boleh melebihi 10 objek termasuk slider tail dan pengulangan.** [Spinner](/wiki/Hit_object/Spinner) adalah pengecualian.
```

File: wiki/Ranking_Criteria/osu!catch/id.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+- **Setidaknya 125 ms wajib dibiarkan di antara circle/slider dan awal dan akhir dari [spinner](/wiki/Hit_object/Spinner).** Ini untuk memastikan keterbacaan.
+
+#### Pedoman
+
+- **[Hyperdash](/wiki/Gameplay/Hyperdash) tidak dapat digunakan pada [drop](/wiki/Hit_object/Juice_stream#drop) individu dan/atau perulangan slider.**
+- **[Hyperdash](/wiki/Gameplay/Hyperdash) yang [basic-snapped](/wiki/Gameplay/Dash_snapping#basic-snapped) tidak boleh digunakan secara berurutan saat menggunakan ketukan beat yang berbeda.** Contohnya, hyperdash 1/1 diikuti oleh hyperdash 1/2.
+- **[Hyperdash](/wiki/Gameplay/Hyperdash) yang [higher-snapped](/wiki/Gameplay/Dash_snapping#higher-snapped) tidak boleh diikuti [antiflow](/wiki/Beatmapping/Antiflow) [Dash](/wiki/Gameplay/Dash) dengan gap yang lebih rendah dari 250 ms.**
+- **[Edge dash](/wiki/Gameplay/Edge_dash) hanya dapat digunakan secara tunggal (tidak berhubungan dengan [Dash](/wiki/Gameplay/Dash) yang lain atau [Hyperdash](/wiki/Gameplay/Hyperdash)).**
+- **[Combos](/wiki/Beatmapping/Combo) tidak boleh melebihi 16 objek termasuk tail slider dan pengulangan.** [Spinner](/wiki/Hit_object/Spinner) adalah pengecualian.
```
Comment (hikikuman) [inline wiki/Ranking_Criteria/osu!catch/id.md:159]:
```suggestion
- **[Kombo](/wiki/Beatmapping/Combo) tidak boleh melebihi 16 objek termasuk tail slider dan pengulangan.** [Spinner](/wiki/Hit_object/Spinner) adalah pengecualian.
```

**Review by hikikuman — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**